### ISSUE TITLE
Unable to set a user for an active Scan

### ISSUE THREAD

**Issue by michaelcourcy** [zaproxy/zaproxy#1776]
Hi 

I'm using ZAP 2.4.1 on a debian  and I can't set a context nor a user for an active scan.

The combo for selecting the user remain disabled. But the context exist and the user also. 

On the previous session I was able to select it, I don't understand this change in the behaviour.

Setting the forced user mode don't change anything.

Watching the log I get this error everytime I load the "active scan" window.
```
2111045 [AWT-EventQueue-0] ERROR org.zaproxy.zap.extension.ascan.PolicyAllCategoryPanel  - Cannot locate configuration source null.policy
org.apache.commons.configuration.ConfigurationException: Cannot locate configuration source null.policy
	at org.apache.commons.configuration.AbstractFileConfiguration.load(AbstractFileConfiguration.java:259)
	at org.apache.commons.configuration.AbstractFileConfiguration.load(AbstractFileConfiguration.java:238)
	at org.apache.commons.configuration.AbstractHierarchicalFileConfiguration.load(AbstractHierarchicalFileConfiguration.java:184)
	at org.zaproxy.zap.utils.ZapXmlConfiguration.<init>(Unknown Source)
	at org.zaproxy.zap.extension.ascan.PolicyManager.loadPolicy(Unknown Source)
	at org.zaproxy.zap.extension.ascan.PolicyManager.getPolicy(Unknown Source)
	at org.zaproxy.zap.extension.ascan.PolicyAllCategoryPanel$3.actionPerformed(Unknown Source)
	at javax.swing.JComboBox.fireActionEvent(JComboBox.java:1258)
	at javax.swing.JComboBox.contentsChanged(JComboBox.java:1332)
	at javax.swing.JComboBox.intervalRemoved(JComboBox.java:1352)
	at javax.swing.AbstractListModel.fireIntervalRemoved(AbstractListModel.java:179)
	at javax.swing.DefaultComboBoxModel.removeAllElements(DefaultComboBoxModel.java:174)
	at javax.swing.JComboBox.removeAllItems(JComboBox.java:771)
	at org.zaproxy.zap.extension.ascan.PolicyAllCategoryPanel.reloadPolicies(Unknown Source)
	at org.zaproxy.zap.extension.ascan.CustomScanDialog.getPolicyAllCategoryPanel(Unknown Source)
	at org.zaproxy.zap.extension.ascan.CustomScanDialog.init(Unknown Source)
	at org.zaproxy.zap.extension.ascan.ExtensionActiveScan.showCustomScanDialog(Unknown Source)
	at org.zaproxy.zap.extension.stdmenus.PopupMenuActiveScanCustom.performAction(Unknown Source)
	at org.zaproxy.zap.view.popup.PopupMenuItemSiteNodeContainer.performAction(Unknown Source)
	at org.zaproxy.zap.view.popup.PopupMenuItemHistoryReferenceContainer.performHistoryReferenceActions(Unknown Source)
	at org.zaproxy.zap.view.popup.PopupMenuItemHistoryReferenceContainer.performActions(Unknown Source)
	at org.zaproxy.zap.view.popup.PopupMenuItemHttpMessageContainer$PerformActionsActionListener.actionPerformed(Unknown Source)
	at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
	at javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2346)
	at javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:402)
	at javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:259)
	at javax.swing.AbstractButton.doClick(AbstractButton.java:376)
	at javax.swing.plaf.basic.BasicMenuItemUI.doClick(BasicMenuItemUI.java:833)
	at javax.swing.plaf.basic.BasicMenuItemUI$Handler.mouseReleased(BasicMenuItemUI.java:877)


```

**Comment by thc202** [comment]
Is the starting point included in the context?
(I'm getting "Not Found" when trying to access the screenshots.)

I'll raise another issue to fix the error logged (it's unrelated to context/user selection).

**Comment by michaelcourcy** [comment]
I think you're right it's probably unrelated.

Your hint was right I didn't set properly the "Include in context/URL Regexs" : 

I set http://172.17.0.43/ and I try to make an active scan on http://172.17.0.43 (without the trailing slash). Once I set \Qhttp://172.17.0.43\E.* everything work.

Sorry for the screenshots the link to my dropbox didnt' work properly I delete them.

I close the issue, thanks.
